paper.md

```diff
@@ -17,6 +17,16 @@ toc: true
 
 This paper proposes element access with bounds checking to `std::mdspan` via `at()` member functions.
 
+# Revision history
+
+## R1
+
+Update wording to align with [@P2821R5]
```

Comment:
Should we maybe more specific?

````
Update wording to align with [@P2821R5]:

- Bump `__cpp_lib_mdspan` instead of adding `__cpp_lib_mdspan_at`
- Mark `at()` as freestanding deleted and adjust freestanding comments of header and class
````